tasks/vmaas_sync/repo_based.go

```diff
@@ -38,18 +38,19 @@ func getCurrentRepoBasedInventoryIDs() ([]mqueue.InventoryAID, error) {
 	return inventoryAIDs, nil
 }
 
-func getRepoBasedInventoryIDs(repos []string) ([]mqueue.InventoryAID, error) {
-	var ids []mqueue.InventoryAID
+func getRepoBasedInventoryIDs(repos []string) ([]mqueue.InventoryAccount, error) {
+	var ids []mqueue.InventoryAccount
 	if len(repos) == 0 {
 		return ids, nil
 	}
 
 	err := database.Db.Table("system_repo sr").
 		Joins("JOIN repo ON repo.id = sr.repo_id").
 		Joins("JOIN system_platform sp ON  sp.rh_account_id = sr.rh_account_id AND sp.id = sr.system_id").
+		Joins("JOIN rh_account ra ON ra.id = sp.rh_account_id").
 		Where("repo.name IN (?)", repos).
 		Order("sp.rh_account_id").
-		Select("distinct sp.inventory_id, sp.rh_account_id").
+		Select("distinct sp.inventory_id, sp.rh_account_id, ra.org_id").
```

Comment:
I see it was there before, I just wonder why the `distinct` is needed

Reply:
Let's ping pepa :smile:

Reply:
Mim authored this one, he will know. Looks like safety check or corner case handle. Do you think it can be a problem?

Reply:
not a problem, just curious. Distinct might be slowing this down a bit

Reply:
Better call @MichaelMraka.

Reply:
@psegedy This should be fine.